### PR TITLE
Feature/5145 Change order status validation to case insensitive

### PR DESCRIPTION
**Pull request by sam-roberts-bjss**
Story: [AB#5145](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5145)
Task: [AB#9123](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/9123)

### CODE REVIEW

File: tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderStatusTests.cs [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        [Test]
+        public void FromId_UnknownOrderStatusId_ReturnsNull()
+        {
+            var actual = OrderStatus.FromId(10);
+            actual.Should().BeNull();
+        }
+
+        [TestCase("Complete")]
+        [TestCase("complete")]
+        [TestCase("comPLete")]
```
Comment (RuthBerry96) [inline tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderStatusTests.cs:42]: Do you not want tests for the order being incomplete?

Comment (sam-roberts-bjss) [reply tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderStatusTests.cs:42]: This test was really just to cover the FromName method & I used the complete as an example. Didn't feel the need to add the incomplete one. Though you have highlighted that the name of the method is wrong. Would you like me to add an incomplete test as well, don't mind either way?

Comment (RuthBerry96) [reply tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderStatusTests.cs:42]: Ahhh, sorry thats my misunderstanding of the test. Dw

Comment (sam-roberts-bjss) [reply tests/NHSD.BuyingCatalogue.Ordering.Domain.UnitTests/OrderStatusTests.cs:42]: No apology needed, it was the right question to ask 👍

**Pull request merged**